flask_admin/static/vendor/multi-level-dropdowns-bootstrap/README.md

```diff
@@ -0,0 +1,61 @@
+# Bootstrap 4 Multiple Level Dropdown
```

Comment:
Do you think there is need to add this README file to static folder?

Reply:
I went through some of the other vendor packages and there have README's so I just left it there.

Reply:
Ok